src/navigation/NavBar.js

```diff
@@ -26,77 +25,73 @@ const NavLink = ({
   const href = `#${to}`
   const active = exact ? hash === href : hash && hash.startsWith(href)
   const anchorClass = classnames({
-    'bg-white-10': active,
+    'bg-white-10 navbar-item-active': active,
     'o-50 no-pointer-events': disabled
-  }, ['dib db-l pv3 white no-underline f5 hover-bg-white-10 tc'])
+  }, ['dib db-l pt2 pb3 pv3-l white no-underline f5 hover-bg-white-10 tc bb bw2 bw0-l b--navy'])
   const svgClass = classnames({
     'o-100': active,
     'o-50': !active
   }, ['fill-current-color'])
 
   return (
     // eslint-disable-next-line jsx-a11y/anchor-is-valid
-    <a href={disabled ? null : href} className={anchorClass} style={{ borderLeft: active ? '5px solid rgba(201, 210, 215, .4)' : '' }} role='menuitem' title={children}>
-      <span className={`dib ${open ? 'dt-l' : ''}`}>
-        <span className={`dib dtc-l v-mid ${open ? 'pl3 pl5-l' : 'ph3'}`} style={{ width: 50 }}>
+    <a href={disabled ? null : href} className={anchorClass} role='menuitem' title={children}>
+      <div className='db ph2 pv1'>
+        <div className='db'>
           <Svg width='50' className={svgClass} />
-        </span>
-        <span className={`${open ? 'dib dtc-l' : 'dn'} pl2 pl3-l pr3 tl-l v-mid `}>
+        </div>
+        <div className={`${active ? 'o-100' : 'o-50'} db f6 tc montserrat ttu fw1 `}>
           {children}
-        </span>
-      </span>
+        </div>
+      </div>
     </a>
   )
 }
 
-export const NavBar = ({ t, open, onToggle }) => {
+export const NavBar = ({ t }) => {
   const codeUrl = 'https://github.com/ipfs-shipyard/ipfs-webui'
   const bugsUrl = `${codeUrl}/issues`
   const gitRevision = process.env.REACT_APP_GIT_REV
   const revisionUrl = `${codeUrl}/commit/${gitRevision}`
   return (
     <div className='h-100 fixed-l flex flex-column justify-between' style={{ overflowY: 'auto', width: 'inherit' }}>
```

Comment:
this container shouldn't be position fixed now, as it just leads to a lot of problems in safari and also container madness for no reason now (I think we had it because of the expand/collapse)

Reply:
@rafaelramalho19 -- I can't find a way to remove the position:fixed while also preserving scrolling behavior and keeping the footer at the bottom of the screen. Any suggestions? (Would this be simpler as an additional PR on top of this one?)